semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -344,7 +476,7 @@ private static String getOperationFromDbQuery(String query) {
             event, getAttributeKeysForRedisTableIndex()));
   }
 
-  private static Optional<String> getOtelDbOperation(Event event) {
+  public static Optional<String> getOtelDbOperation(Event event) {
```

Comment:
why are we exposing it?

Reply:
because we don't have fallbacks or specific conditions for Cassandra DB Operation as of now so we are exposing same function there.

Reply:
And even for some other backends if we add in future, this should be useful.

Reply:
The point was around calling this method `getOtelDbOperation`, if you do see its usage outside, call it `getDbOperation` which would allow for extending it with attributes for other formats, ow you would need `getOTDbOperation` and so on.

Reply:
ohh. makes sense. Done.